tutorials/04-switching-physics-engines.md

```diff
@@ -0,0 +1,99 @@
+\page switchphysicsengines Physics engines
+
+This tutorial describes how to switch between physics engines in Ignition Physics when using Ignition Gazebo.
+Ignition Physics enables Ignition Gazebo to choose flexibly what physics engine to use at runtime.
+By default, Ignition Gazebo uses the [DART](https://dartsim.github.io/) physics engine.
+
+Downstream developers may also integrate other physics engines by creating new Ignition Physics engine plugins.
+See [Ignition Physics](https://ignitionrobotics.org/api/physics/3.0/tutorials.html)'s tutorials to learn how to integrate a new engine.
```

Comment:
Consider using `\ref tutorials "Ignition Physics"` instead of the ignitionrobotics.org URL so doxygen can autogenerate the appropriate URL corresponding to the current version.